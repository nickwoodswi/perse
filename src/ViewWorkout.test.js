import React from 'react';
import { render } from '@testing-library/react';
import ViewWorkout from './ViewWorkout';
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

test('renders ViewWorkout component', () => {
    render(<MemoryRouter><ViewWorkout /></MemoryRouter>);
});

it('ViewWorkout renders the UI as expected', () => {
    const tree = renderer
        .create(<MemoryRouter><ViewWorkout /></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});