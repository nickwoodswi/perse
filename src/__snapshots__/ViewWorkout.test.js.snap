// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewWorkout renders the UI as expected 1`] = `
<div
  className="view-workouts"
>
  <div
    className="view-workouts-headline"
  >
    <h2>
      VIEW SCHEDULED WORKOUTS BY ATHLETE:
    </h2>
  </div>
  <select
    className="athlete-selector"
    onChange={[Function]}
  >
    <option>
      SELECT ATHLETE
    </option>
  </select>
  <div
    className="athlete-workouts"
  />
</div>
`;
